transitioning-issues-from-cg-to-wg.md

```diff
@@ -0,0 +1,38 @@
+# Transition process from proposals to spec for the Immersive Web
+
+The Immersive Web Working Group and Community Group work together to develop the specifications for building Augmented Reality and Virtual Reality applications on the web. New proposals will be developed in the community group and transition to the working group when they are ready. This document aims to outline the process for taking an idea all the way from inception through to publishing in the Immersive Web groups.
+
+## 1. Proposing an Idea
+
+Ideas should first be proposed as [issues in the proposals repo](https://github.com/immersive-web/proposals/issues). Here they can be raised to the attention of the group, and be discussed and rationalised in the context of the Immersive Web work.  Proposals may also be raised to discuss in the community group teleconferences.
+
+The intent at this stage is to raise awareness, gauge interest and get feedback on the idea or potential approaches to solutions. 
+
+## 2. Creating a Repo
+
+Once there is clear interest from multiple parties in exploring an issue, the Community Group chair can create a repository to discuss (or move one into the Immersive-Web organisation, if the proposer had one already).  
+
+The repository for a proposal enables issue tracking, and branching.  In particular, any issues raised in the initial discussion in the "proposals" repository should be filed in the new repository.
+
+The best place to start with any new proposal is **[an explainer](https://w3ctag.github.io/explainers)** to provide a quick introduction to why the feature is needed and what it does. It should be written for a non-technical audience to provide a fast introduction for non-experts to understand.
+
+At this stage it's still okay for their just to be a single contributor managing and controlling the contents of the proposal. It is also okay to push changes straight to master without needing peer review.
```

Comment:
their => there
or, preferably, "...it's still okay for a single contributor to manage and control the contents..."